content/en/docs/concepts/packages/_index.md

```diff
@@ -77,7 +77,7 @@ within the module root:
 
     cue mod init [module name]
 
-The module name is required if a packages within a module should be able
+The module name is required if a package within a module should be able
```

Comment:
Thanks, change looks good. But we could simplify the language a bit further.

Perhaps:

> The module name is required if a package within the module needs to import another package within the module.